internal/checker/cloudformation/rules/secrets/verify_lambda_environment_have_no_hard_coded_secrets.rego

```diff
@@ -0,0 +1,51 @@
+# METADATA
+# title: "Verify Lambda environment have no hard coded secrets."
+# description: "This policy checks AWS Lambda functions to ensure that no hard-coded secrets are present in their environment variables."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html
+# - https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-function.html
+# custom:
+#   ID: CB_CFT_35
+#   Severity: High
+package lib.cloudformation.CB_CFT_35
+import future.keywords.in
+
+resource := ["AWS::Lambda::Function", "AWS::Serverless::Function"]
+
```

Comment:
update is_valid
is_valid {
    some resources in input.Resources
    resources.Type  in resource
}